batch: replace the deprecated registerTempTable() to reslove the compile warning

```
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/tobe/CheckNull.scala:56: warning: method re
gisterTempTable in class Dataset is deprecated (since 2.0.0): Use createOrReplaceTempView(viewName) instead.
[WARNING]     df.registerTempTable("t1")
[WARNING]        ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/tobe/CheckNull.scala:57: warning: method re
gisterTempTable in class Dataset is deprecated (since 2.0.0): Use createOrReplaceTempView(viewName) instead.
[WARNING]     df.registerTempTable("t2")
[WARNING]        ^
```

You can reproduce the issue by compiling java modules with `mvn clean package`.

--- a/java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/end2end/unsafe/TestDateUdf.scala
+++ b/java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/end2end/unsafe/TestDateUdf.scala
@@ -79,7 +79,7 @@ class TestDateUdf extends SparkTestSuite {
     ))
 
     val t1 = spark.createDataFrame(spark.sparkContext.makeRDD(data), schema)
-    t1.registerTempTable("t1")
+    t1.createOrReplaceTempView("t1")
     sess.registerTable("t1", t1)
 
     val sqlText = "SELECT col1, day(col1), dayofmonth(col1), dayofweek(col1) FROM t1"
@@ -103,7 +103,7 @@ class TestDateUdf extends SparkTestSuite {
     ))
 
     val t1 = spark.createDataFrame(spark.sparkContext.makeRDD(data), schema)
-    t1.registerTempTable("t1")
+    t1.createOrReplaceTempView("t1")
     sess.registerTable("t1", t1)
 
     val sqlText ="""

--- a/java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/end2end/unsafe/TestTimestampUdf.scala
+++ b/java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/end2end/unsafe/TestTimestampUdf.scala
@@ -45,7 +45,7 @@ class TestTimestampUdf extends SparkTestSuite {
       StructField("col1", TimestampType)))
 
     val t1 = spark.createDataFrame(spark.sparkContext.makeRDD(data), schema)
-    t1.registerTempTable("t1")
+    t1.createOrReplaceTempView("t1")
     sess.registerTable("t1", t1)
 
     val sqlText = "SELECT col1, year(col1), month(col1), day(col1) FROM t1"
@@ -69,7 +69,7 @@ class TestTimestampUdf extends SparkTestSuite {
     ))
 
     val t1 = spark.createDataFrame(spark.sparkContext.makeRDD(data), schema)
-    t1.registerTempTable("t1")
+    t1.createOrReplaceTempView("t1")
     sess.registerTable("t1", t1)
 
     val sqlText ="""

--- a/java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/end2end/unsafe/TestWindowWithoutSelect.scala
+++ b/java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/end2end/unsafe/TestWindowWithoutSelect.scala
@@ -45,7 +45,7 @@ class TestWindowWithoutSelect extends SparkTestSuite {
     val t1 = spark.createDataFrame(spark.sparkContext.makeRDD(data), schema)
 
     sess.registerTable("t1", t1)
-    t1.registerTempTable("t1")
+    t1.createOrReplaceTempView("t1")
 
     val sqlText =
       """      select

